components/k8s-model-server/images/releaser/components/workflows.libsonnet

```diff
@@ -313,40 +372,30 @@
                 mirrorVolumeMounts: true,
               }],
             ),  // deploy-tf-serving
+            buildTemplate(
+              "deploy-tf-serving-gpu",
+              deploy_tf_serving_gpu_command,
+              [
+                {
```

Comment:
Are these environment variables correct? Why do you need to use docker in docker when you deploy TF serving?

Reply:
We needed because we were building the image. Removing now.